src/Algebra/Bundles.agda

```diff
@@ -504,7 +504,7 @@ record SemiringWithoutOne c ℓ : Set (suc (c ⊔ ℓ)) where
 
   open NearSemiring nearSemiring public
     using
-    ( _≉_; +-rawMagma; +-magma; +-unitalMagma; +-semigroup
```

Comment:
Why are you dropping the reexport of `_≉_`?

Reply:
Thanks for the speedy eyes: it's dropped because Agda complained about ambiguous imports, presumably arising from the `open Setoid setoid public` addition, which brings these things into scope already (I think?). Removing them made the typechecker happy, so... (see previous about my finding this corner of the library *very* arcane)... I thought that I should be happy too...

Reply:
OK, this looks correct, then.